admin/HowToBuildDocs.md

```diff
@@ -0,0 +1,26 @@
+# How to Build Shield Documentation
+
+## Requirements
+
+- Python3
+- pip
+
+## Installation
+
+```console
+pip3 install mkdocs
+pip3 install mkdocs-material
+pip3 install mkdocs-git-revision-date-localized-plugin
+```
+
+## Build the Docs
+
+```consolse
+mkdocs build
+```
+
+## See the Docs
+
+```consolse
+mkdocs serve
+```
```

Comment:
Command ` mkdocs gh-deploy` is also good for manual implementation if there is a problem. Of course, I think it needs administrator access.

Reply:
Added.